README.md

```diff
@@ -16,6 +16,7 @@ Creates a RDS instance, security_group, subnet_group and parameter_group
 | availability\_zone | The availability zone where you want to launch your instance in | string | `""` | no |
 | backup\_retention\_period | How long do you want to keep RDS backups | string | `"14"` | no |
 | default\_parameter\_group\_family | Parameter group family for the default parameter group, according to the chosen engine and engine version. Defaults to mysql5.7 | string | `"mysql5.7"` | no |
+| enabled\_cloudwatch\_logs\_exports | List of log types to enable for exporting to CloudWatch logs. You can check the available log types per engine in the [AWS RDS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.html#USER_LogAccess.Procedural.UploadtoCloudWatch). Defaults to nothing. | list | `[]` | no |
```

Comment:
Can you make sure this documentation is also added in full to the output's description? We use [terraform-docs](https://github.com/segmentio/terraform-docs) to generate the inputs/outputs documentation.

Reply:
you probably mean the "variable's description" 😛

Reply:
lol yes 😅

Reply:
@duboisph @iuriaranda updated the variable description.